hr_worked_days_hourly_rate/README.rst

```diff
@@ -0,0 +1,63 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+Worked Days Hourly Rate
```

Comment:
There's still missing the upper line. See https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst for a sample